aws-ts-apigateway-eventbridge/README.md

```diff
@@ -0,0 +1,83 @@
+# API Gateway V1 to EventBridge
+
+[![Deploy with Pulumi](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/aws-ts-apigateway-eventbridge)
+
+This example demonstrates an API Gateway V1 integration with EventBridge and Lambda that also validates request bodies (using an API Gateway model) and returns a custom HTTP response.
+
+![Diagram showing an API Gateway-EventBridge integration](https://user-images.githubusercontent.com/274700/149042008-ce1bedd3-f795-409f-b8ab-57d1ff78c259.png)
+
+## Prerequisites
+
+1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/).
+1. [Install Node.js](https://www.pulumi.com/docs/intro/languages/javascript/).
+1. Configure your [AWS credentials](https://www.pulumi.com/docs/intro/cloud-providers/aws/setup/).
+
+### Deploying the App
+
+1. Clone this repo, change to this directory, then create a new [stack](https://www.pulumi.com/docs/intro/concepts/stack/) for the project:
+
+    ```bash
+    pulumi stack init
+    ```
+
+1. Specify an AWS region to deploy into:
+
+    ```bash
+    pulumi config set aws:region us-west-2
+    ```
+
+1. Install Node dependencies and run Pulumi:
+
+    ```bash
+    npm install
+    pulumi up
+    ```
+
+1. In a few moments, the API Gateway instance service will be up and running and its public URL emitted as a Pulumi [stack output](https://www.pulumi.com/docs/intro/concepts/stack/#outputs).
+
+    ```bash
+    ...
+    Outputs:
+        url: "https://andchh8hg8.execute-api.us-west-2.amazonaws.com/dev"
+    ```
+
+1. Verify the deployment with `curl`:
+
+    With invalid POST data:
+
+    ```bash
+    curl --data '{"some-invalid-property-name": "Chris"}' --header "Content-Type: application/json" "$(pulumi stack output url)/uploads"
+
+    HTTP/2 400
+    {"message": "Invalid request body"
```

Comment:
I'm guessing this is the full message so it should close?
```suggestion
    {"message": "Invalid request body"}
```

Reply:
Ah yes, bad paste on my part. Thanks!